org.Cataclysm.DDA.Tiles.yml

```diff
@@ -0,0 +1,54 @@
+id: "org.Cataclysm.DDA.Tiles"
```

Comment:
`Cataclysm.org` doesn't resolve.

But you said

> The domain used for the application ID is [controlled by the application developers](https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository) and the [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

Reply:
maybe you meant `org.cataclysmdda` ?

![image](https://github.com/user-attachments/assets/bf908086-c497-412a-b937-3c5817b21dc6)

Reply:
Yes, I accidentally made a mistake